### DISCUSSION TITLE
Tested with T540c : works well ! Thank you

### DISCUSSION THREAD

**Discussion by john821** [cagnulein/qdomyos-zwift#499] [Show and tell]
tested T540C + Raspberry pi w : work well ! thank you

**Comment by cagnulein** [comment]
thanks for the feedback!

**Reply by john821** [reply]
In order to support your Project, I bought your App on Google Play ( but I'm using my rPi, not my phone )

**Reply by cagnulein** [reply]
thanks @john821 ! remember to rate the app on the store, it means a lot to me!

**Comment by Badaboom38** [comment]
Bonsoir. Je viens de télécharger l'appli sur mon tel afin de pouvoir mettre de connecter mon tapis T540C sur zwift mais ça ne fonctionne pas. Pouvez-vous m'aider ?
En vous remerciant

**Reply by cagnulein** [reply]
Hi are you using 2 devices: one for zwift and one for qz?if so which devices are they?
